Resources/ServerInfo/Guidebook/Science/Cyborgs.xml

```diff
@@ -0,0 +1,66 @@
+<Document>
+  # Cyborgs
+  Cyborgs are man-machine hybrids, purpose-built by the [textlink="robotics"  link="Robotics"] division of science for various duties across the station. Bound by [color=cyan]silicon laws[/color] which they follow strictly, and religiously, they serve the crew unconditionally. [italic](Most of the time.)[italic]
+
+  ## Basic Components
+  <Box>
+    <GuideEntityEmbed Entity="CyborgEndoskeleton" Caption="Cyborg Endoskeleton"/>
+    <GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
+  </Box>
+  All cyborgs are built on a endoskeleton, which can be crafted at an [color=#a4885c]Exosuit Fabricator[/color]. Further assembly steps can be followed by [color=#a4885c]examining[/color] it. Once built, further upgrades, such as additional tools, and longer battery life can be done with modules and a better power cell, respectively.
+
+  In addition, newly built cyborgs need a [color=#a4885c]brain[/color] able to interface with the electronic body. There are two main types of such, both of which are able to speak independently, should the chassis be decommissioned, exchanged, or otherwise [color=red]suffer an unscheduled dissassembly[/color];
```

Comment:
```suggestion
  Newly built cyborgs need a [color=#a4885c]brain[/color] able to interface with the electronic body. There are two main types, both of which are able to speak independently, should the chassis be decommissioned, exchanged, or otherwise suffer an unscheduled disassembly.
```